.coderabbit.yaml

```
@@ -0,0 +1,127 @@
+  auto_review:
+    enabled: true
+    drafts: false
+chat:
+  auto_reply: true
+
+
+# config.yaml for CodeRabbit
+project:
+  name: django_project
+
+language:
```

yamllint

[error] duplication of key "language" in mapping

(key-duplicates)